jgrapht-demo/src/main/java/org/jgrapht/demo/DirectedGraphDemo.java

```diff
@@ -0,0 +1,118 @@
+/* 
+ * Copyright (C) 2008-2018,  Minh Van Nguyen &lt; nguyenminh2@gmail.com&gt;
```

Comment:
Can you replace this old class header by the new one? Just copy-paste the class header from another class, and change the year to 2008-2018 and give the copyright to Minh Van Nguyen